test/Identity.FunctionalTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
Actually, can we add a comment here that explains why we did it, and links to the test failure issue?

Reply:
Updated with link and some words